exchanges/graphcache/src/cacheExchange.ts

```diff
@@ -86,12 +86,10 @@ export const cacheExchange = (opts?: CacheExchangeOpts): Exchange => ({
   forward,
   client,
 }) => {
-  if (!opts) opts = {};
```

Comment:
Good shout on removing this!

Reply:
To be fair, I've just moved it 😆  but typically transpilers will jump through more hoops in the generated code with default params rather than reassigning the argument. It's a minor details though, since we do get the type safety from TypeScript in either cases :+1:

Reply:
`default params` are very expensive size wise, see [this REPL](https://babeljs.io/en/repl#?browsers=&build=&builtIns=false&spec=false&loose=false&code_lz=MYewdgzgLgBAHjAvDAFAbwFAxgJwIZgAmIAtkjAKwA0WMANgKYQQBKBxJNAvgJRIB8MTNkbM2RUuVGt2kgD5yYANgDctHAygBXHGFyyyAKnpMZEkmq4YgA&debug=false&forceAllTransforms=false&shippedProposals=false&circleciRepo=&evaluate=false&fileSize=false&timeTravel=false&sourceType=module&lineWrap=true&presets=env%2Cenv&prettier=false&targets=&version=7.8.7&externalPlugins=)